packages/www/docs/api-reference/vod/upload.mdx

```diff
@@ -83,26 +83,15 @@ axios
     },
     "task": {
         "id": "$TASK_ID",
-        "type": "import",
-        "outputAssetId": "$ASSET_ID",
-        "params": {
-          "import": {
-            "uploadedObjectKey": "directUpload/$PLAYBACK_ID/source"
-          }
-        },
-        "createdAt": 1652813126031,
-        "status": {
-            "phase": "pending",
-            "updatedAt": 1652813126031
-        }
     }
 }
 ```
 
-You can use the returned `task` object to monitor the progress of processing the
-asset after the contents have been uploaded. Check the
-[Retrieve a Task](/docs/api-reference/vod/list-tasks#retrieve-a-task) API for
-more information.
+You can use the returned `asset` object to monitor the status of the asset after
+the contents have been uploaded. Check the
+[Retrieve a Asset](/docs/api-reference/vod/list#retrieve-an-asset) API for more
+information. You can also use the returned `task.id` to track the partial
+progress using the Task API.
```

Comment:
Do we have any docs on the task APIs? If so, should link them here as well.

If not, we should have that at some point 😬

Reply:
This is the docs for what we have now: https://github.com/livepeer/livepeer-studio-docs/blob/main/docs/references/vod/list-tasks.mdx

I can add these changes there as well 😀